core/domain/services/admin/events/data/ConfirmDeletion.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace EventEspresso\core\domain\services\admin\events\data;
+
+use EE_Admin_Page;
+use EE_Error;
+use EED_Batch;
+use EEH_URL;
+use EventEspresso\admin_pages\events\form_sections\ConfirmEventDeletionForm;
+use EventEspresso\core\services\orm\tree_traversal\NodeGroupDao;
+
+/**
+ * Class ConfirmDeletion
+ *
+ * Description
+ *
+ * @package     Event Espresso
+ * @author         Mike Nelson
+ * @since         $VID:$
+ *
+ */
+class ConfirmDeletion
+{
+    /**
+     * @var NodeGroupDao
+     */
+    private $dao;
+
+    /**
+     * ConfirmDeletion constructor.
+     * @param NodeGroupDao $dao
+     */
+    public function __construct(
+        NodeGroupDao $dao
+    )
+    {
+
+        $this->dao = $dao;
+    }
+
+    /**
+     * @since $VID:$
+     * @param \Events_Admin_Page $admin_page
+     * @throws EE_Error
+     * @throws \EventEspresso\core\exceptions\InvalidDataTypeException
+     * @throws \EventEspresso\core\exceptions\InvalidInterfaceException
+     * @throws \EventEspresso\core\exceptions\UnexpectedEntityException
+     * @throws \InvalidArgumentException
+     * @throws \ReflectionException
```

Comment:
plz import all classes (ie use `use` statements) so that all dependencies are listed at the top of the file

Reply:
good catch thanks